core/server/services/invitations/accept.js

```diff
@@ -0,0 +1,36 @@
+const common = require('../../lib/common');
+const models = require('../../models');
+const security = require('../../lib/security');
+
+function accept(invitation) {
+    const data = invitation.invitation[0];
```

Comment:
Same here about passing the function nicer arguments, and having the call site deal with whatever format